src/browser/ui/dom/components/ReactDOMInput.js

```diff
@@ -66,7 +67,7 @@ var ReactDOMInput = ReactClass.createClass({
 
   render: function() {
     // Clone `this.props` so we don't mutate the input.
```

Comment:
This comment does feel superfluous, with the comment at:
https://github.com/facebook/react/pull/3349/files#diff-7697f9038a7012efcea5287199198866R30